docs/serverless/03-05-supported-webhooks.md

```diff
@@ -71,3 +71,13 @@ It checks the following conditions for these CRs:
 
       - **spec.auth.type** must be set to either `key` (SSH key) or `basic` (password or token)
       - **spec.auth.secretName** must not be empty
+
+## Admission webhook
+
+There is also an additional, admission webhook that is only triggered in the scenario when you switch in the runtime from one registry to another.
```

Comment:
```suggestion
There is also an additional admission webhook that is only triggered in the scenario when you switch in the runtime from one registry to another.
```

Reply:
Without a comma, it somehow sounds like there are some admission webhooks and there is one more, which I would prefer to avoid. How about simply saying: `There is also an admission webhook `?